poetry/json/__init__.py

```diff
@@ -31,7 +31,9 @@ def validate_object(obj, schema_name):  # type: (dict, str) -> List[str]
     for error in validation_errors:
         message = error.message
         if error.path:
-            message = "[{}] {}".format(".".join(error.path), message)
+            message = "[{}] {}".format(
+                ".".join(str(x) for x in error.absolute_path), message
```

Comment:
Is there any reason why you use `absolute_path` instead of `path`?

Reply:
as far as i remember because both are an alias and the name i use is clearer what it represents.